tests/unit_tests.c

```diff
@@ -672,6 +708,9 @@ int unit_tests(int argc, char *argv[])
     #endif
     test_wolfTPM2_KeyBlob(TPM_ALG_RSA);
     test_wolfTPM2_KeyBlob(TPM_ALG_ECC);
+    test_wolfTPM2_GetEK(TPM_ALG_NULL);
+    test_wolfTPM2_GetEK(TPM_ALG_RSA);
```

Comment:
This test is causing the `examples/run_examples.sh`
`./examples/keygen/create_primary -rsa -eh -idevid -keep` to fail because it already exists... You need to fix the `create_primary.c` example to not "fail" if it already exists...